src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -3065,40 +3337,209 @@ namespace detail
                     }
                 }
 
+                // handle volume dependent fields
+                if (vol_dep)
+                {
+                    // make volume into a field
+                    Node &volumes_field = fields_dest["volume"];
```

Comment:
also add the prefix name here.

We only need to calc the volume once. Since this is in the loop over fields, as is it could happen multiple times unnecessarily.

To solve this, you can check if the field was already created on subsequent trips around the loop.